app/gui2/src/App.vue

```diff
@@ -1,13 +1,18 @@
 <script setup lang="ts">
+import HelpScreen from '@/components/HelpScreen.vue'
 import { provideAppClassSet } from '@/providers/appClass'
 import { provideGuiConfig, type GuiConfig } from '@/providers/guiConfig'
 import { useSuggestionDbStore } from '@/stores/suggestionDatabase'
+import type { ApplicationConfig } from '@/util/config'
 import ProjectView from '@/views/ProjectView.vue'
 import { onMounted, toRef } from 'vue'
 
 const props = defineProps<{
   config: GuiConfig
```

Comment:
This `config: GuiConfig` now seems to be redundant with `appConfig`, as it effectively repeats the same parameters.